### PR TITLE
Refactor app bar line

### PR DESCRIPTION
**Pull request by DevKarenC**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,15 +1,15 @@
 import React, { Component } from 'react';
 import Dropdown from './components/Dropdown';
-import USTopTenPositive from './components/USTopTenPositive';
-import USPositiveHistory from './components/USPositiveHistory';
+import USTopTenPositive from './components/unitedstates/USTopTenPositive';
+import USPositiveHistory from './components/unitedstates/USPositiveHistory';
```
Comment (AlanC5) [inline src/App.js:4]: Are these components only ever going to be used for US?

File: src/App.js [hunk 2]
```diff
@@ -33,118 +33,12 @@ class App extends Component {
...
-        positiveIncrease: stateData.positiveIncrease,
-        stateName: stateData.state,
-      };
-      return positiveIncreaseArray;
-    });
-    return currentDataCopy
-      .sort((a, b) => b.positiveIncrease - a.positiveIncrease)
-      .slice(0, size);
-  }
-
```
Comment (AlanC5) [inline src/App.js:147]: Nice clean up here!

Comment (DevKarenC) [reply src/App.js:147]: Thanks!

File: src/App.js (hunk 1 above)
Comment (AlanC5) [reply src/App.js:4]: Seems like the components can be country agnostic. Any country specific details can be passed in as props?

**Review by AlanC5 — APPROVED** [review]
LGTM otherwise

**Pull request merged**